src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -1,14 +1,15 @@
 module.exports = {
 
   async afterCreate(event) {
-    console.log(event)
-    const { idea_card: idea, author: commentAuthor, text: commentText, createdAt: timeCreated } = event.result;
+    const { author: commentAuthor, text: commentText, createdAt: timeCreated } = event.result;
     const commentId = event.result.id
-    console.log(idea)
+    const comment = await strapi.entityService.findOne("api::comment.comment", commentId, {
+      populate: ["idea_card"],
+    })
     
     await strapi.entityService.create('api::notification.notification', {
       data: {
-        Title: commentAuthor + " commented on " + idea?.ideaName,
+        Title: commentAuthor + " commented on " + comment?.idea_card?.ideaName,
```

Comment:
Is this a bug fix?

Reply:
Can we fix it in a separate PR? It's easier for IdeaSpace to review if it's in a smaller PR.

Reply:
OK. Found it while adding tests.

Reply:
Reverted. We can fix it, as we are working on notification system changes